data/presets/climbing/area.json

```diff
@@ -0,0 +1,15 @@
+{
+    "icon": "temaki-climbing",
+    "fields": [
+        "name"
+    ],
+    "geometry": [
+        "relation",
+        "point"
```

Comment:
https://wiki.openstreetmap.org/wiki/Climbing#Climbing_areas only mentions relations and https://taginfo.openstreetmap.org/tags/climbing=area#overview shows little usage on points (and ways).

I think we should only promote the relation here and use the #1590 for points. 

A cleanup via MapRoulette might be good as a parallel effort.

Reply:
I agree that areas should only be represented as relations. I changed this now.